addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -227,9 +231,27 @@ export class AddPageTemplatePreview extends Component {
                 holderEl.classList.add("o_ready");
             };
             adjustHeight();
+            if (this.props.isCustom) {
+                this.adaptCustom(wrapEl);
+            }
         });
     }
 
+    adaptCustom(wrapEl) {
```

Comment:
Either change the name to `adaptCustomTemplate`, or add a doc string above it or a comment before its call (above after `adjustHeight`) to describe better what this function does :wink: